Source/ConversationV1/Models/DialogNode.swift

```diff
@@ -102,15 +102,17 @@ public struct DialogNode {
      - parameter context: The context (if defined) for the dialog node.
      - parameter metadata: The metadata (if any) for the dialog node.
      - parameter nextStep: The next step to execute following this dialog node.
-     - parameter actions: The actions for the dialog node.
+     - parameter created: The timestamp for creation of the dialog node.
      - parameter title: The alias used to identify the dialog node.
+     - parameter updated: The timestamp for the most recent update to the dialog node.
+     - parameter actions: The actions for the dialog node.
      - parameter nodeType: How the dialog node is processed.
      - parameter eventName: How an `event_handler` node is processed.
      - parameter variable: The location in the dialog context where output is stored.
 
      - returns: An initialized `DialogNode`.
     */
-    public init(dialogNodeID: String, created: String, updated: String, description: String? = nil, conditions: String? = nil, parent: String? = nil, previousSibling: String? = nil, output: [String: JSON]? = nil, context: [String: JSON]? = nil, metadata: [String: JSON]? = nil, nextStep: DialogNodeNextStep? = nil, actions: [DialogNodeAction]? = nil, title: String? = nil, nodeType: String? = nil, eventName: String? = nil, variable: String? = nil) {
+    public init(dialogNodeID: String, description: String, conditions: String, parent: String, previousSibling: String, output: [String: JSON], context: [String: JSON], metadata: [String: JSON], nextStep: DialogNodeNextStep, created: String, title: String, updated: String? = nil, actions: [DialogNodeAction]? = nil, nodeType: String? = nil, eventName: String? = nil, variable: String? = nil) {
```

Comment:
Looks like the `created`, `updated`, and `actions` properties have just been moved around a bit in the `staging/conversation` branch.